src/renderer/views/Subscriptions/Subscriptions.js

```diff
@@ -1,492 +1,56 @@
     'ft-card': FtCard,
-    'ft-button': FtButton,
-    'ft-icon-button': FtIconButton,
-    'ft-flex-box': FtFlexBox,
-    'ft-element-list': FtElementList,
-    'ft-channel-bubble': FtChannelBubble
+    'ft-flex-box': FtFlexBox
   },
   data: function () {
     return {
-      isLoading: false,
-      dataLimit: 100,
-      videoList: [],
-      errorChannels: [],
-      attemptedFetch: false,
+      currentTab: 'videos'
     }
   },
-  computed: {
-    backendPreference: function () {
-      return this.$store.getters.getBackendPreference
-    },
-
-    backendFallback: function () {
-      return this.$store.getters.getBackendFallback
-    },
-
-    currentInvidiousInstance: function () {
-      return this.$store.getters.getCurrentInvidiousInstance
-    },
-
-    hideWatchedSubs: function () {
-      return this.$store.getters.getHideWatchedSubs
-    },
-
-    useRssFeeds: function () {
-      return this.$store.getters.getUseRssFeeds
-    },
-
-    activeVideoList: function () {
-      if (this.videoList.length < this.dataLimit) {
-        return this.videoList
-      } else {
-        return this.videoList.slice(0, this.dataLimit)
-      }
-    },
-
-    activeProfile: function () {
-      return this.$store.getters.getActiveProfile
-    },
-    activeProfileId: function () {
-      return this.activeProfile._id
-    },
-
-    cacheEntriesForAllActiveProfileChannels() {
-      const entries = []
-      this.activeSubscriptionList.forEach((channel) => {
-        const cacheEntry = this.$store.getters.getSubscriptionsCacheEntriesForOneChannel(channel.id)
-        if (cacheEntry == null) { return }
-
-        entries.push(cacheEntry)
-      })
-      return entries
-    },
-    videoCacheForAllActiveProfileChannelsPresent() {
-      if (this.cacheEntriesForAllActiveProfileChannels.length === 0) { return false }
-      if (this.cacheEntriesForAllActiveProfileChannels.length < this.activeSubscriptionList.length) { return false }
-
-      return this.cacheEntriesForAllActiveProfileChannels.every((cacheEntry) => {
-        return cacheEntry.videos != null
-      })
-    },
-
-    historyCache: function () {
-      return this.$store.getters.getHistoryCache
-    },
-
-    activeSubscriptionList: function () {
-      return this.activeProfile.subscriptions
-    },
-
-    hideLiveStreams: function() {
-      return this.$store.getters.getHideLiveStreams
-    },
-
-    hideUpcomingPremieres: function () {
-      return this.$store.getters.getHideUpcomingPremieres
-    },
-
-    fetchSubscriptionsAutomatically: function() {
-      return this.$store.getters.getFetchSubscriptionsAutomatically
-    },
-  },
   watch: {
-    activeProfile: async function (_) {
-      this.isLoading = true
-      this.loadVideosFromCacheSometimes()
+    currentTab(value) {
+      // Save last used tab, restore when view mounted again
+      sessionStorage.setItem('Subscriptions/currentTab', value)
     },
   },
   mounted: async function () {
```

Comment:
By @absidue 
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/5f0b027c-2616-402f-bc70-198b937ee0c3)


```suggestion
  created: async function () {
```